orderportal/form.py

```diff
@@ -434,6 +454,63 @@ def post(self, iuid):
         self.see_other("form_edit", saver.doc["_id"])
 
 
+class FormEmailSender(FormMixin, RequestHandler):
+    """
+    Send an email to all users who have made orders for the form.
+    The email will contain a link to a survey.
+    """
+
+    @tornado.web.authenticated
+    def post(self, iuid):
+        self.check_admin()
+
+        form = self.get_form(iuid)
+        # Check if form is enabled
+        if form["status"] != constants.ENABLED:
+            self.see_other("form", form["_id"], error="Form is not enabled.")
+            return
+
+        # Check if survey link is set
+        link = form.get("survey_link", None)
+        if not link:
+            self.see_other("form", form["_id"], error="No survey link set.")
+            return
+
+        # Get recipients
+        recipients = self.get_order_emails(form)
+        owner = form.get("owner", None)
+        recipients.append(owner)  # Add form owner to the list of recipients
+
+        # Prepare email content
+        site = settings.get("SITE_NAME") or "OrderPortal"
+        url = settings.get("BASE_URL") # can't be none
+        form_title = form.get("title") # can't be none
+        text = """The owners of {site} ({url}) would like to get feedback from you regarding {form_title}.
+
+The survey is available at the following link:
+{link}.
+
+    
+Yours sincerely,
+The {site} administrators.
```

Comment:
Better! I don't think we need "the owners", I think we could go for just the order portal name immediately. I suggest the following: 
```suggestion
        text = """The {site} ({url}) would like to get your feedback regarding your order {form_title}.

The {site} asks that you please fill in the following survey: {link}. 

Thank you in advance!
The {site} administrators.
```